src/ctia/graphql/schemas.clj

```diff
@@ -135,6 +136,14 @@
                                        sighting/sighting-order-arg
                                        p/connection-arguments)
                        :resolve (res/search-entity-resolver :sighting)}
+    :target_record    {:type    TargetRecordType
+                       :args    search-by-id-args
+                       :resolve (res/entity-by-id-resolver :target-record)}
```

Comment:
@ereteog I'm stuck. I've been trying to figure it out, but I'm really stuck.

For some reason the last commit is breaking things and I'm not sure exactly what I'm missing (TargetRecord should almost be like AssetMapping, but as soon I uncomment the highlighted lines, I can't start the server anymore, it fails with:

    2021-02-01 18:24:43,380 ERROR (async-dispatch-2) [puppetlabs.trapperkeeper.internal] - Error during service start!!! graphql.AssertException: type Verdict not found in schema at graphql.Assert.assertTrue(Assert.java:41) at graphql.schema.SchemaUtil.resolveTypeReference(SchemaUtil.java:244) at graphql.schema.GraphQLList.replaceTypeReferences(GraphQLList.java:45) at graphql.schema.SchemaUtil.resolveTypeReference(SchemaUtil.java:248) at graphql.schema.GraphQLFieldDefinition.replaceTypeReferences(GraphQLFieldDefinition.java:69) at graphql.schema.SchemaUtil.resolveTypeReferencesForFieldsContainer(SchemaUtil.java:228) at graphql.schema.SchemaUtil.replaceTypeReferences(SchemaUtil.java:212) at graphql.schema.GraphQLSchema$Builder.build(GraphQLSchema.java:300) at graphql.schema.GraphQLSchema$Builder.build(GraphQLSchema.java:289) at ctia.schemas.graphql.helpers$eval52786$new_schema__52791$fn__52792$fn52793__52803$fn__52804.invoke(helpers.clj:488)
        at ctia.schemas.graphql.helpers$eval52786$new_schema__52791$fn__52792$fn52793__52803.invoke(helpers.clj:486)
        at clojure.lang.AFn.applyToHelper(AFn.java:154)
        at clojure.lang.AFn.applyTo(AFn.java:144)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:408)
        at ctia.schemas.core$eval34569$resolve_with_rt_ctx__34574$fn__34575.invoke(core.clj:158)
        at ctia.schemas.core$eval34569$resolve_with_rt_ctx__34574.invoke(core.clj:152)
        at ctia.schemas.graphql.helpers$eval52831$new_graphql__52836$fn__52837$fn52838__52848$fn__52849.invoke(helpers.clj:502)
        at ctia.schemas.graphql.helpers$eval52831$new_graphql__52836$fn__52837$fn52838__52848.invoke(helpers.clj:500)
        at clojure.lang.AFn.applyToHelper(AFn.java:154)
        at clojure.lang.AFn.applyTo(AFn.java:144)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:408)
        at ctia.schemas.core$eval34569$resolve_with_rt_ctx__34574$fn__34575.invoke(core.clj:158)
        at ctia.schemas.core$eval34569$resolve_with_rt_ctx__34574.invoke(core.clj:152)
        at ctia.http.server_service_core$eval86545$start__86550$fn__86551.invoke(server_service_core.clj:40)
        at ctia.http.server_service_core$eval86545$start__86550.invoke(server_service_core.clj:24)
        at ctia.http.server_service$reify__86646$service_fnk__12275__auto___positional$reify__86658.start(server_service.clj:21)
        at puppetlabs.trapperkeeper.services$eval12073$fn__12087$G__12063__12090.invoke(services.clj:9)
        at puppetlabs.trapperkeeper.services$eval12073$fn__12087$G__12062__12094.invoke(services.clj:9)
        at puppetlabs.trapperkeeper.internal$eval22325$run_lifecycle_fn_BANG___22332$fn__22333.invoke(internal.clj:196)
        at puppetlabs.trapperkeeper.internal$eval22325$run_lifecycle_fn_BANG___22332.invoke(internal.clj:179)
        at puppetlabs.trapperkeeper.internal$eval22354$run_lifecycle_fns__22359$fn__22360.invoke(internal.clj:229)
        at puppetlabs.trapperkeeper.internal$eval22354$run_lifecycle_fns__22359.invoke(internal.clj:206)
        at puppetlabs.trapperkeeper.internal$eval22982$build_app_STAR___22991$fn$reify__23003.start(internal.clj:605)
        at puppetlabs.trapperkeeper.internal$eval23030$boot_services_for_app_STAR__STAR___23037$fn__23038$fn__23040.invoke(internal.clj:631)
        at puppetlabs.trapperkeeper.internal$eval23030$boot_services_for_app_STAR__STAR___23037$fn__23038.invoke(internal.clj:629)
        at puppetlabs.trapperkeeper.internal$eval23030$boot_services_for_app_STAR__STAR___23037.invoke(internal.clj:623)
        at clojure.core$partial$fn__5841.invoke(core.clj:2630)
        at puppetlabs.trapperkeeper.internal$eval22399$initialize_lifecycle_worker__22410$fn__22411$fn__22562$state_machine__19592__auto____22587$fn__22590.invoke(internal.clj:249)
        at puppetlabs.trapperkeeper.internal$eval22399$initialize_lifecycle_worker__22410$fn__22411$fn__22562$state_machine__19592__auto____22587.invoke(internal.clj:249)
        at clojure.core.async.impl.ioc_macros$run_state_machine.invokeStatic(ioc_macros.clj:978)
        at clojure.core.async.impl.ioc_macros$run_state_machine.invoke(ioc_macros.clj:977)
        at clojure.core.async.impl.ioc_macros$run_state_machine_wrapped.invokeStatic(ioc_macros.clj:982)
        at clojure.core.async.impl.ioc_macros$run_state_machine_wrapped.invoke(ioc_macros.clj:980)
        at clojure.core.async$ioc_alts_BANG_$fn__19839.invoke(async.clj:421)
        at clojure.core.async$do_alts$fn__19770$fn__19773.invoke(async.clj:288)
        at clojure.core.async.impl.channels.ManyToManyChannel$fn__14327$fn__14328.invoke(channels.clj:95)
        at clojure.lang.AFn.run(AFn.java:22)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
        at clojure.core.async.impl.concurrent$counted_thread_factory$reify__14196$fn__14197.invoke(concurrent.clj:29)
        at clojure.lang.AFn.run(AFn.java:22)
        at java.base/java.lang.Thread.run(Thread.java:832)